bigbluebutton-html5/private/config/settings.yml

```diff
@@ -138,6 +138,7 @@ public:
       captureSharedNotesByDefault: false
       sendInvitationToAssignedModeratorsByDefault: false
       breakoutRoomLimit: 16
+      allowUploadNewDocsInBreakouts: false
```

Comment:
We want to limit the display of upload/manage docs. From the name here it implies you can still show the modal just can't upload *new*... We also want to not allow changing presentation visibility, export with annotations etc.
Perhaps naming `allowDocsUploadInBreakouts` or `allowDocsManagementInBreakouts` might be generic enough. What do you think?